docs/index.rst

```diff
@@ -83,6 +83,43 @@ Installation
 For more information about supported accelerators and platforms, and for other
 installation options, see the `Install Guide`_ in the project README.
 
+Ecosystem
+---------
+JAX's efficient numerical computing forms the foundation for a growing ecosystem of tools:
+
+.. card-carousel:: 3
+
+   .. card:: :material-regular:`schema;2em` Neural Networks
+
+      - `Flax <https://flax.readthedocs.io/>`__
+      - `Haiku <https://dm-haiku.readthedocs.io/>`__
+      - `Equinox <https://docs.kidger.site/equinox/>`__
+
+   .. card:: :material-regular:`area_chart;2em` Bayesian Modeling
+
+      - `NumPyro <https://num.pyro.ai/>`__
+      - `PyMC <https://www.pymc.io/>`__
+      - `tinygp <https://tinygp.readthedocs.io/>`__
+
+   .. card:: :material-regular:`track_changes;2em` Optimization
+
+      - `Diffrax <https://docs.kidger.site/diffrax/>`__
+      - `Optax <https://optax.readthedocs.io/>`__
```

Comment:
@jakevdp I can add short descriptions to some of these if you think this makes sense. (otherwise, what is Optax vs JAXOpt, for example)

Reply:
Done, thanks!